docs/spec/ibc/ibc.md

```diff
@@ -0,0 +1,80 @@
+# IBC Specification
+
+*This is a living document and should be edited as the IBC spec and 
+implementation change*
+
+## Engineering Philosophy
+
+The goal is to get the simplest implementation working end-to-end first. Once
+the simplest and most insecure and most feature-less use-case is implemented
+we can start adding features. Let's get to the end-to-end process first though.
+
+
+## MVP One
+
+The initial implementation of IBC will include just enough for simple coin 
+transfers between chains, with safety features such as ACK messages being added 
+later.
+
+### IBC Module
+
+* handles message processing
+
+```golang
+type IBCOutMsg struct {
+  IBCTransfer
+}
+
+type IBCInMsg struct {
+  IBCTransfer
+}
+
+type IBCTransfer struct {
+  Destination sdk.Address
```

Comment:
dont we need a destination ChainID ?

Reply:
Yes, eventually we will need to add a couple of more fields to `IBCTransfer`. The goal for the first iteration is to keep everything as simple as possible so that we can get to an end-to-end process the fastest. Once we have an end-to-end process we'll extend everything and build up everything around it.

Reply:
But what good is an MVP of IBC if we can't send to another chain? Maybe I'm not understanding something ...

Reply:
We will assume that only two zones(hubs) are connected to each other directly, so the destination chain id is determined implicitly. We will add most of security functionalities(destchain checking, lightclient..) on the mvp2.